### PR TITLE
Fixed paintbrush getting damaged in creative

### PR DESCRIPTION
**Pull request by ron-popov**
Fixed issue #3834 , fixed the issue by checking if the player holding the brush is in creative mode, if he is then there is no change to the brush's durability.

### CODE REVIEW

File: common/buildcraft/core/item/ItemPaintbrush_BC8.java [hunk 1]
```diff
@@ -195,7 +195,12 @@ public boolean useOnBlock(World world, BlockPos pos, IBlockState state, Vec3d hi
             if (result == EnumActionResult.SUCCESS) {
                 ParticleUtil.showChangeColour(world, hitPos, colour);
                 SoundUtil.playChangeColour(world, pos, colour);
-                usesLeft--;
+
+                boolean isHolderCreative = player.isCreative();
+                if (!isHolderCreative) {
```
Comment (AlexIIL) [inline common/buildcraft/core/item/ItemPaintbrush_BC8.java]: Bit minor, but as there's no uses for this, can you inline ````isHolderCreative```` into the if statement?

Comment (ron-popov) [reply common/buildcraft/core/item/ItemPaintbrush_BC8.java]: sure

**Comment by AlexIIL** [comment]
Thanks :)

**Comment by AEnterprise** [comment]
Can you confirm your discord username and discrim here for the contributor role on discord?

**Comment by ron-popov** [comment]
My Discord name is "DirtyAxe"

**Comment by FrancisDaPugtato** [comment]
:D Thanks for fixing it! :D

**Pull request merged**